src/sentry/consumers/__init__.py

```diff
@@ -267,6 +267,9 @@ def ingest_events_options() -> list[click.Option]:
         "static_args": {
             "consumer_type": "feedback-events",
         },
+        "dlq_topic": Topic.INGEST_FEEDBACK_EVENTS_DLQ,
+        "dlq_max_invalid_ratio": 0.01,
+        "dlq_max_consecutive_count": 1000,
```

Comment:
Mind removing these 2 lines? We experimented with various values but now I think we should just skip these (across the board not specific to your use case)

Reply:
Sure, should we remove from the other ones too?

Reply:
Yes we probably should. But I'll double check and do it as a follow up -- let's just focus on the ingest-feedback-events-dlq here.

Reply:
yep, sounds good

Reply:
similarly, I noticed ingest-events consumer doesn't have a dlq_topic. Not something to worry about for this PR though

Reply:
I'm pretty sure it does have now. I just added it. You might need to rebase/merge master to see it.